receptorctl/receptorctl/cli.py

```diff
@@ -143,9 +143,10 @@ def traceroute(ctx, node):
 @click.argument('node')
 @click.argument('service')
 @click.option('--raw', '-r', default=False, is_flag=True, help="Set terminal to raw mode")
-def connect(ctx, node, service, raw):
+@click.option('--tlsclient', type=str, default="", help="TLS client used when connecting to remote node")
```

Comment:
Maybe "TLS client config name" to make it more obvious what this parameter actually is?

Reply:
fixed